src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -41,6 +41,15 @@ class AttributeMetadata implements AttributeMetadataInterface
      */
     public $maxDepth;
 
+    /**
+     * @var string|null
+     *
+     * @internal This property is public in order to reduce the size of the
+     *           class' serialized representation. Do not access it. Use
+     *           {@link getSerializedName()} instead.
+     */
+    public $serializedName;
```

Comment:
Is it still really necessary to make this property public? We do that everywhere in validator (and I copied this behavior in Serializer), but I'm not sure that it's still accurate.
Anyway, keep it here for consistency.